### PR TITLE
Update the selection on create; proper defaults for IronMultiSelectableBehavior

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: test/multi.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
       });
 
       test('has sane defaults', function() {
-        assert.equal(s.selectedValues, undefined);
+        assert.deepEqual(s.selectedValues, []);
```
Comment (bicknellr) [inline test/multi.html]: I argue that this is a more sane default. :P

Comment (valdrinkoshi) [inline test/multi.html]: \o/ indeed!

**Comment by valdrinkoshi** [comment]
LGTM, are you sure you want to merge this into `master` though?

**Comment by bicknellr** [comment]
> are you sure you want to merge this into master though?

Hmm, no, I'll put this in a new PR merged into the 2.0-preview branch.

**Pull request closed without merging**